bigchaindb/backend/schema.py

```diff
@@ -99,3 +103,21 @@ def init_database(connection=None, dbname=None):
     create_database(connection, dbname)
     create_tables(connection, dbname)
     create_indexes(connection, dbname)
+
+
+def validate_if_exists_asset_language(tx_body):
+    data = tx_body['asset'].get('data', {})
+
+    if data and 'language' in data:
+
+        language = data.get('language')
+        backend = bigchaindb.config['database']['backend']
+
+        if backend == 'mongodb' and language not in VALID_LANGUAGES:
```

Comment:
Similar to my previous remark, I'd have much more peace of mind if this was written
```python
if (backend == 'mongodb') and (language not in VALID_LANGUAGES):
```